helpers/disabledSummaries.ts

```diff
@@ -0,0 +1,23 @@
+export function isDiscordSummaryDisabled(cacheKey: string): boolean {
+  const disabledList = process.env.DISABLED_DISCORD_SUMMARY || "";
+
+  if (!disabledList) {
+    return false;
+  }
+
+  // Extract the key part after "discord-summary:"
+  const keyPart = cacheKey.replace("discord-summary:", "");
+
+  // Split the disabled list by comma and check if the key is in the list
+  const disabledKeys = disabledList.split(",").map((key) => key.trim());
```

Comment:
In rare cases there might be a comma in the title (part of a request key in `DISABLED_DISCORD_SUMMARY`). this may split the array incorrectly.

Reply:
it also broke if the title had a $ in it due to escape chars :)

have addressed this now.